graphql/schema/types/config.graphql

```diff
@@ -89,12 +89,6 @@ input ConfigGeneralInput {
   excludes: [String!]
   """Array of file regexp to exclude from Image Scans"""
   imageExcludes: [String!]
-  """Scraper user agent string"""
-  scraperUserAgent: String
-  """Scraper CDP path. Path to chrome executable or remote address"""
-  scraperCDPPath: String
-  """Whether the scraper should check for invalid certificates"""
-  scraperCertCheck: Boolean!
```

Comment:
As above, these should be left in. `scraperCertCheck` should be made optional. For these three fields, the server should only change the configuration value if the fields are present. Mark as deprecated as needed.